### PR TITLE
T shcha/cicd

### PR DESCRIPTION
**Pull request by tshcha**
### Purpose

Source code for CICD.
I'd like it to be included into RC1.3.3_Revit2009 as soon as possible because now I can't run all stages of Jenkinsfile script together to test it.
I need to run only first stage to clone repo, then copy .bat files, CS.prop, .env, .version there manually,  checkout .version file before run other 3 stages.
After Jenkinsfile, 3 .bat scripts, updated CS.props, .env and .version files are cloned from repo it will be possible to run all 4 stages together.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

Phil XIa

Aaron Tang

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -0,0 +1 @@
+0.0.3
```
Comment (philxia1) [inline .version]: according to current version in artifactory, we need to reset it to 0.0.5

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+BRANCH_NAME='master'
```
Comment (philxia1) [inline .env]: why we need this file?

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        bat batFolder + '\\' + 'BuildSolutions.bat'
+      } catch(err) {
+        echo "Dynamo or DynamoRevit Build Failed: $err"
+        currentBuild.result = 'FAILURE'
+        throw err
+      }
+    }
+
+    if (branch == 'master' || branch.indexOf('release') == 0) {
+      stage('Deployment to Artifactory') {
```
Comment (philxia1) [inline Jenkinsfile]: please remove the "|| branch.indexOf('release') == 0" as we only support the master branch right now.

File: .env (hunk 2 above)
Comment (tshcha) [reply .env]: I've completed according to revit-model-browser. Branch name is set in external file there.
I'll avoid using of this file for d4r and set branch name in Jenkinsfile directly.

File: src/scripts/ci/Deploy.bat [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    IF EXIST %folder% (
+        rmdir /s /q %folder%
+    ) 
+
+    call mkdir %folder%
+    call xcopy /y /i /s "../Dynamo/bin/AnyCPU/release" "./%folder%/release"
+    call xcopy /y /i /s "../Dynamo/bin/AnyCPU/debug" "./%folder%/debug"
+    call xcopy /y /i /s "../Dynamo/bin/AnyCPU/debug" "./%folder%/slow_debug"
+    call "C:\Program Files\7-Zip\7z.exe" a -tzip "%folder%.zip" ".\%folder%\release" ".\%folder%\debug" ".\%folder%\slow_debug"
+    call "C:\DeployTool\art.exe" upload "%folder%.zip" "team-revit-generic/revit-d4r/" --url=https://art-bobcat.autodesk.com/artifactory --user= --password=
```
Comment (philxia1) [inline src/scripts/ci/Deploy.bat]: why we still need to call art.exe here? I thought we need to remove it.

**Pull request merged**